input/pagecontent/change_log.md

```diff
@@ -1,3 +1,62 @@
+**The following changes occurred between [STU 3 publication](https://hl7.org/fhir/us/mcode/STU3/) (October 2023) and the STU 4 ballot (May 2024). For a history of previous changes, please see the prior change logs in the [appropriate versions](http://hl7.org/fhir/us/mcode/history.html).**
+
+### Addition of non-TNM Staging Profiles
+
+mCODE 4.0.0-ballot includes additional non-TNM staging profiles. These staging profiles are especially important for pediatric oncology.
+
+* [NeuroblastomaINSStage]
+* [NeuroblastomaINRGSSRiskAssessment]
+* [WilmsTumorStage]
+* [RhabdomyosarcomaClinicalGroupStage]
+
+### Modification of Lymphoma Stage Profile
+
+Lymphoma staging can include many modifiers, such as bulky status, extranodal involvement, splenic involvement, and symptom indicators. As a result, components for these modifiers were added to the [LymphomaStage] profile to better describe the lymphoma stage.
+
+### Addition of Risk Assessment Profiles
+
+mCODE STU3 did not include any profiles for risk assessments, so there was no indication on how to use mCODE to represent different risk assessments. To address this issue, an abstract, generic profile was created for risk assessment, and specific risk assessment profiles were added for illustrative purposes:
+
+* [CancerRiskAssessment]
+* [RhabdomyosarcomaRiskAssessment]
+* [ALLRiskAssessment]
+
+### Addition of Histology Profiles
+
+mCODE 4.0.0-ballot includes three new profiles for histologic reporting. [HistologicBehaviorAndType] reports the morphology and type of the tumor using ICD-O-3 codes and the proper suffix, while [HistologicGrade] reports the grade of the tumor. Both profiles are referenced in a summary with the [TumorMorphology] profile.
+
+* [HistologicBehaviorAndType]
+* [HistologicGrade]
+* [TumorMorphology]
```

Comment:
The title of [TumorMorphology] is 'Tumor Morphology Report'. 
I recommend changing the computable name of the profile to align with the title, since the misalignments require cognitive energy to resolve.  It is a report on Tumor Morphology.